src/modify_resources.py

```diff
@@ -20,7 +20,7 @@
 ip_pattern = r'\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b'
 
 # Container blacklist for container names whose placements should not move
-container_blacklist = ['ovn-controller', 'minion', 'ovs-vswitchd', 'ovsdb-server', 'etcd']
+container_blacklist = ['ovn-controller', 'minion', 'ovs-vswitchd', 'ovsdb-server', 'etcd', 'hantaowang/lumbersexual']
```

Comment:
Should maintain separate blacklists: one for the Quilt blacklist and one for application specific services .